ooyala_player/ooyala_player.py

```diff
@@ -153,18 +152,16 @@ def student_view(self, context):
         # Skin file path according to block type
         if hasattr(self, 'lightchild_block_type'):
             json_config_url = resource_url(OoyalaPlayerLightChildBlock.lightchild_block_type, SKIN_FILE_PATH)
-            bit_movin_player_url = resource_url(OoyalaPlayerLightChildBlock.lightchild_block_type, BIT_MOVIN_PLAYER_PATH)
```

Comment:
Are these changes related to translations? If not, revert these lines.

Reply:
nope